### DISCUSSION TITLE
Error 1101 after updating to V 0.1.2

### DISCUSSION THREAD

**Discussion by spaceroam** [microfeed/microfeed#64] [Q&A] [answered]
I recommended Microfeed to a friend, and after she installed it, everything seemed fine in the dashboard. She could edit and publish blog posts and even receive them through RSS. However, she couldn't access the blog page and kept getting an error 1101. I then updated my Microfeed from version 0.1.1 to 0.1.2, but after deploying it to Cloudflare, my blog pages also showed the same error. Like my friend, my admin backend was still working fine. When I rolled back to the previous deployment, the blog page worked normally again. Has anyone else experienced this issue? Is it a problem with version 0.1.2, or is there another reason for the error?

**Comment by wenbinf** [comment]
Let's figure this out together - 

1) Login your Cloudflare account.

2) Go to your latest Pages deployment, click Functions tab, and click "Begin log stream" -

<img width="1279" alt="Screenshot 2023-03-17 at 8 36 44 PM" src="https://user-images.githubusercontent.com/1719237/226082664-59fcf30a-3a5d-465e-ab0e-54837d4bf7e4.png">

3) Now visit the page that throws out Error 1101 on your site. You should be able to see logs from the above log stream.

Let me know what you see there.

**Reply by 19-10-9** [reply]
it says
"exceptions": [
    {
      "name": "Error",
      "message": "Dynamic require of \"./4a37b559bc564e0183c71344c123120e18c08761-web_feed.html\" is not supported",
      "timestamp": 1679124287839
    }

**Reply by wenbinf** [reply]
I can reproduce this issue for a new microfeed instance.

Looks like it's related to esbuild issues:
* https://answers.netlify.com/t/netlify-serverless-typescript-functions-fail-with-dynamic-requires/52379/12
* https://github.com/evanw/esbuild/issues/1921

Cloudflare might change esbuild settings or something else. I will dig more over the weekends.

**Comment by wenbinf** [comment]
Another thing to do to help troubleshot - 

Could you email your Web Item's template code to support@microfeed.org ?
You can find the code at /admin/settings/code-editor/?type=themes&theme=custom#webItem -

<img width="1299" alt="Screenshot 2023-03-17 at 8 49 54 PM" src="https://user-images.githubusercontent.com/1719237/226083504-27baa5aa-16bd-4419-ab19-62e0e4b1d2a8.png">

**Reply by 19-10-9** [reply]
Hi, I'm the friend mentioned above by spaceroam. 
I'm using microfeed v.0.1.2 and can't access any pages except for the admin page - the Code Editor also shows an 1101 error.

**Comment by 19-10-9** [comment]
Someone posted a similar issue on Cloudflare in Jan. I think she/he was using a previous version of microfeed.
https://community.cloudflare.com/t/how-to-troubleshoot-1101-error/451749

**Comment by wenbinf** [comment, marked as answer]
@spaceroam @19-10-9 

Just merged this PR https://github.com/microfeed/microfeed/pull/65 to fix.

Please sync your forked repo to the latest version and re-deploy.

**Reply by spaceroam** [reply]
It's working perfectly fine now! Thank you so much for fixing the issue so quickly. Really appreciate your work.

**Reply by 19-10-9** [reply]
Thank you, it works for me as well.
